src/App.js

```diff
@@ -860,6 +861,16 @@ function App() {
                             remoteControlSessionElement,
                         ]}
                     />
+                    <Route
+                        path={'/achievements/'}
```

Comment:
```suggestion
                        path={'/achievements'}
```

Perhaps this doesn't matter, but should it just be `/achievements` instead?

Reply:
Our rout paths simply aren't standardized in any way in this regard. Some have a trailing slash, some don't, but it doesn't seem to make any difference in loading pages.